src/superio/nuvoton/nct5104d/superio.c

```diff
@@ -106,6 +110,69 @@ static void route_pins_to_uart(struct device *dev, bool to_uart)
 	pnp_write_config(dev, 0x1c, reg);
 }
 
+static void enable_gpio_io_port(struct device *dev, bool enable_wdt1)
+{
+	u8 reg, reg1;
+	u16 io_base_address;
+	u8 uartc_enabled, uartd_enabled;
+
+	pnp_write_config(dev, 0x07, NCT5104D_GPIO_WDT);
+
+	/* 
+		Devictree always sets LDN 8 CR30.0 bit (WDT1 enable)
+		See if user really wants Watchdog Timer 1 to be enabled
+	*/ 
+
+	if(!enable_wdt1)
+	{
+		reg = pnp_read_config(dev, 0x30);
+		pnp_write_config(dev, 0x30, reg & 0xFE);
+	}
+
+	/*  
```

Comment:
Comments should be like:

```
/* this is some comment
 * this is a second line of comment
 */
```

We do not use indentation inside comments like that